src/components/enterprise/dashboard/sidebar/DashboardSidebar.jsx

```diff
@@ -71,7 +74,7 @@ class DashboardSidebar extends React.Component {
             <p>
               To request more benefits or specific courses,
               {' '}
-              {this.renderLearningCoordinatorHelpText(enterpriseName, enterpriseEmail)}.
+              {this.renderLearningCoordinatorHelpText()}.
```

Comment:
Minor refactor to just call the function without passing arguments, since those can be be retrieved directly from the `pageContext` prop in the function.

Reply:
This makes that function less unit-testable (should we ever decide unit tests are something we want. Just calling this out. No change needed.

Reply:
That is definitely true, and something to definitely keep in mind. Always open to feedback on how to make things more testable 😃 

My mindset around testing functions that render basic things, like in this case, has typically been that it'd be covered in a snapshot test where you the test renders the component in whatever state necessary to render said basic things and perform a snapshot test on it there. If it's not something interactive (e.g.,  testing button clicks), and just worried about it rendering the correct thing, snapshots have mostly been fine in the past.

Though, that said, snapshots are somewhat dangerous in that you have to really pay attention to what's failing in the snapshot before updating it to make sure its changes are what you expect 🤷‍♂